Makefile

```diff
@@ -91,25 +92,44 @@ linux:
 # test runs the test suite.
 test:
 	@echo "==> Testing ${NAME}"
-	@go test -timeout=30s -parallel=20 -tags="${GOTAGS}" ${GOFILES} ${TESTARGS}
+	@go test -timeout=30s -parallel=20 -tags="${GOTAGS}" ${GOPKGS} ${TESTARGS}
 .PHONY: test
 
 # test-race runs the test suite.
 test-race:
 	@echo "==> Testing ${NAME} (race)"
-	@go test -timeout=60s -race -tags="${GOTAGS}" ${GOFILES} ${TESTARGS}
+	@go test -timeout=60s -race -tags="${GOTAGS}" ${GOPKGS} ${TESTARGS}
 .PHONY: test-race
 
 # test without VCR
 test-full:
 	@echo "==> Testing ${NAME} with VCR disabled"
 	@env \
 		VCR_DISABLE=1 \
-		go test -timeout=60s -parallel=20 ${GOFILES} ${TESTARGS}
+		go test -timeout=60s -parallel=20 ${GOPKGS} ${TESTARGS}
 .PHONY: test-full
 
 # update fixtures default service ID
 fix-fixtures:
 	@echo "==> Updating fixtures"
 	@$(CURRENT_DIR)/scripts/fixFixtures.sh ${FASTLY_TEST_SERVICE_ID}
 .PHONY: fix-fixtures
+
+# lists improperly-formatted files if they exist
+check-fmt:
+	@gofmt -s -l ${GOFILES}
+	@printf "\nPlease, run 'make fmt'.\n"
```

Comment:
This needs to check for a non-zero exit code from the above command before printing this.